packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -1,7 +1,7 @@
+import type { AuthClient } from './'
 import type { UserAgentApplication as AzureActiveDirectory } from 'msal'
```

Comment:
This is Prettier's formatting, which ESLint doesn't like:
<img width="611" alt="Screen Shot 2021-05-14 at 12 20 25 PM" src="https://user-images.githubusercontent.com/2951/118318873-d5404400-b4ae-11eb-919f-26bd9a3bb84f.png">